source/_integrations/zwave_js.markdown

```diff
@@ -983,18 +983,23 @@ Switching does not require renaming your devices.
 
 4. Configure the Z-Wave JS UI add-on with the added control panel, including setting the location of your Z-Wave device and the network security keys.
 
-5. Add the Z-Wave integration again (even though it is still installed), and uncheck the "Use the Z-Wave JS Supervisor add-on". Enter the correct address for the community add-on in the URL field in the next step.
+5. Add the Z-Wave integration again (even though it is still installed).
```

Comment:
Maybe we should remove step 1 and update the text here to not say to add the Z-Wave integration again? The reconfigure flow will now unload the config entry and stop the add-on during the move from the official add-on to Z-Wave JS UI.